src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -105,8 +106,11 @@ def _handle_global_configuration(config):
         answers['modify_global_prompt'] = should_modify_global_config
     if not config_exists or should_modify_global_config:
         # no config exists yet so configure global config or user wants to modify global config
+        need_to_reset_use_local_config = True if config.use_local_config else False
```

Comment:
Can be simplified `need_to_reset_use_local_config  = config.use_local_config`

Reply:
Good catch. Updated.